mythril/leveldb/state.py

```diff
@@ -43,9 +43,9 @@ class Account(rlp.Serializable):
         ('code_hash', hash32)
     ]
 
-    def __init__(self, nonce, balance, storage, code_hash, db, address):
+    def __init__(self, nonce, balance, storage, code_hash, db, addr):
```

Comment:
Same here, why address -> addr

Reply:
The local name address shadows an imported thingie that's also called address